docs/aca/09-aca-autoscale-keda/index.md

```diff
@@ -103,27 +105,28 @@ az containerapp update `
 --scale-rule-auth "connection=svcbus-connstring" `
 --scale-rule-metadata "topicName=<Your topic name>" `
                         "subscriptionName=<Your topic subscription name>" `
-                        "namespace=<Your service bus namespace>" `
+                        "namespace=$NamespaceName" `
                         "messageCount=10" `
                         "connectionFromEnv=svcbus-connstring"
 ```
 
 What we have done is the following:
```

Comment:
Make this consistent with how we have done for other places where were are sort of explaining the command itself ?https://azure.github.io/aca-dotnet-workshop/aca/04-aca-dapr-stateapi/#4-enable-dapr-for-the-frontend-web-app-and-backend-api-container-apps

![image](https://user-images.githubusercontent.com/8058415/229438210-8237f35c-c2f9-474a-bfce-b157923e936a.png)